config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
what do you think about an endpoint at `manifests/validations#create` instead? that way we are working within the standard rails endpoints